### PR TITLE
Sam

### PR DESCRIPTION
**Pull request by kparks29**

### CODE REVIEW

File: .jshintrc [hunk 1]
```diff
@@ -14,6 +14,7 @@
   "globals": {
     "angular": true,
     "_": true,
+    "$": true,
```
Comment (kparks29) [inline .jshintrc:17]: did you add jquery?

File: gulpfile.js [hunk 2]
```diff
@@ -1,33 +1,31 @@
...
+var shell       = require('gulp-shell');
+var clean       = require('gulp-clean');
+var ngAnnotate  = require('gulp-ng-annotate');
+var usemin      = require('gulp-usemin');
+var uglify      = require('gulp-uglify');
+var minifyCss   = require('gulp-minify-css');
+var runSequence = require('run-sequence');
+var webserver   = require('gulp-webserver');
+var fs          = require('fs');
+var paths       = { src: 'src' };
```
Comment (kparks29) [inline gulpfile.js:17]: any reason for changing this?  it is the same

File: src/app/book/book.js [hunk 3]
```diff
@@ -1,14 +1,48 @@
-(function () {
+(function() {
   'use strict';
 
-  function BookCtrl ($location) {
+  function BookCtrl() {
+    var date       = $('#date');
+    var startTime  = $('#time-start');
```
Comment (kparks29) [inline src/app/book/book.js:6]: you shouldn't mix jquery with angular.  angular already comes with jquery lite
jquery lite is `angular.element('#time-start')`

File: src/app/common/directives.js [hunk 4]
```diff
@@ -1,40 +1,38 @@
-(function () {
+(function() {
```
Comment (kparks29) [inline src/app/common/directives.js:1]: why are you removing the space before the `()` most linters want that space there

File: src/app/common/directives.js [hunk 5]
```diff
@@ -1,40 +1,38 @@
...
     return {
       link: function(scope, elm, attr) {
         scope.$on('$routeChangeSuccess', function() {
           var href = '/' + attr.href.split('/')[2];
+
           if ($location.path() === href) {
             elm.addClass('nav-active');
-          }
-          else {
+          } else {
```
Comment (kparks29) [inline src/app/common/directives.js:12]: this is good practice with the else statements! :+1:

File: src/app/common/directives.js [hunk 6]
```diff
@@ -1,40 +1,38 @@
...
             elm.removeClass('transparent');
           }
         });
       }
     };
   }
 
-
-  angular.module('Directives', [])
+  angular
```
Comment (kparks29) [inline src/app/common/directives.js:34]: not sure this is a best practice, it doesnt hurt anything, but it can become unclear.  just a person preference and the preference of my mentor

File: src/app/contact/contact.css [hunk 7]
```diff
@@ -0,0 +1,47 @@
+.contact {
+  max-width: 750px;
+  margin-right: auto;
+  margin-bottom: 0;
```
Comment (kparks29) [inline src/app/contact/contact.css:4]: combine the three margin statements into 1

File: src/app/contact/contact.css [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+}
+
+.contact label,
+.contact label small {
+  font-weight: 300;
+}
+
+.contact a {
+  color: #006F3E;
+  color: rgba(0,111,62,.8);
```
Comment (kparks29) [inline src/app/contact/contact.css:35]: `color: #006F3E;` should be deleted since it is being overridden in the next line and never used

File: src/app/home/home.html [hunk 9]
```diff
@@ -1,7 +1,15 @@
...
-</ul>
\ No newline at end of file
+<div class="banner clearfix">
+  <h1 class="col-xs-12 text-center">What Kind of Monster Are You?</h1>
+  <img class="img-responsive col-xs-4 col-sm-2" alt="Creature" src="app/assets/images/creature-of-the-black-lagoon.png" />
+  <img class="img-responsive col-xs-4 col-sm-2" alt="Potato" src="app/assets/images/attack-of-the-killer-potato.png" />
+  <img class="img-responsive col-xs-4 col-sm-2" alt="American" src="app/assets/images/american-werewolf.png" />
+  <img class="img-responsive col-xs-4 col-sm-2" alt="Melt" src="app/assets/images/mighty-melt.png" />
+  <img class="img-responsive col-xs-4 col-sm-2" alt="Carnivore" src="app/assets/images/carnivore.png" />
+  <img class="img-responsive col-xs-4 col-sm-2" alt="Blue" src="app/assets/images/big-blue-meanie.png" />
```
Comment (kparks29) [inline src/app/home/home.html:8]: this is not concise, why did you move away from the ng-repeat?

File: src/app/styles/index.css [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+
+/*=========================
+           PAGES
+=========================*/
+@import '../home/home.css';
+@import '../schedule/schedule.css';
+@import '../menu/menu.css';
+@import '../book/book.css';
+@import '../contact/contact.css';
+@import '../about/about.css';
```
Comment (kparks29) [inline src/app/styles/index.css:26]: :+1:

File: .jshintrc (hunk 1 above)
Comment (samjwalston) [inline .jshintrc:17]: jQuery was already in the project. Probably for the bootstrap plugins in to work. Which I don't think you ever used initially. But i set that so i could use jQuery for a date picker plugin which is in the booking controller.

File: gulpfile.js (hunk 2 above)
Comment (samjwalston) [inline gulpfile.js:17]: Other stuff was changed in there when we were testing stuff. For me that is easier to read and the other stuff got removed.

File: src/app/book/book.js (hunk 3 above)
Comment (samjwalston) [inline src/app/book/book.js:6]: Fair enough. Didn't know that angular had some jQuery. jQuery was already installed so I figured I'd use that.

File: src/app/common/directives.js (hunk 4 above)
Comment (samjwalston) [inline src/app/common/directives.js:1]: Style guides I've read before have suggested that. So that is how I've done them in my projects. Didn't know about the linter rules.

File: src/app/contact/contact.css (hunk 8 above)
Comment (samjwalston) [inline src/app/contact/contact.css:35]: Used as a fallback for if rgba is not supported. Legacy browsers.
